dev-go/gox/gox-1.0.1.ebuild

```diff
@@ -7,13 +7,19 @@ inherit go-module
 DESCRIPTION="A dead simple, no frills Go cross compile tool"
 HOMEPAGE="https://github.com/mitchellh/gox"
 SRC_URI="https://github.com/mitchellh/gox/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://dev.gentoo.org/~jsmolic/gox-riscv.patch"
 
 LICENSE="MPL-2.0"
 SLOT="0/${PVR}"
 KEYWORDS="amd64 ~arm64 ~ppc64 ~x86"
```

Comment:
No ~riscv keyword?

Reply:
Could you make 2 commits (one that adds the patch and one that adds ~riscv keyword)

Reply:
My mistake, I now split it into two commits one with the riscv patch added and one with the riscv keyword added